projects/hipfft/clients/tests/CMakeLists.txt

```
@@ -73,6 +73,27 @@ else()
     get_target_property(fftw3_libs FFTW3::fftw3 IMPORTED_LOCATION_RELEASE)
     get_target_property(fftw3f_libs FFTW3::fftw3f IMPORTED_LOCATION_RELEASE)
 
+    # Look for omp (preferred) or thread libraries.  These are not a
+    # hard requirement, but are nice to have to make FFTW run faster.
+    find_library(FFTW_OMP_LIBRARY NAMES fftw3_omp HINTS ${FFTW3_LIBRARY_DIRS})
+    find_library( FFTW_THREADS_LIBRARY fftw3_threads HINTS ${FFTW3_LIBRARY_DIRS} )
+    if( FFTW_OMP_LIBRARY )
```

Maybe set to FALSE before all the logic that follows afterwards?
```suggestion
    set( FFTW_MULTITHREAD FALSE )
    if( FFTW_OMP_LIBRARY )
```

---

I also thinked that but I think it's better not to set it newer to NO because in both in the rocFFT and hipFFT there are just multiple places which set's it TRUE and newer to NO. 
And all the checks then just check if FFTW_MULTITHREAD is set.
If I would start changing it here, it would for clarify be good then to init it in all couple of other places also first to NO. Below is the grepped usage of that variable.


```
lamikr@ub2404:~/own/rock/src/sdk/rocm-libraries/projects/rocfft$ git grep FFTW_MULTITHREAD
clients/tests/CMakeLists.txt:        set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:        set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:        set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:        set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:  set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:if(FFTW_MULTITHREAD)
clients/tests/CMakeLists.txt:  target_compile_options( rocfft-test PRIVATE -DFFTW_MULTITHREAD )
clients/tests/cmake/FindFFTW.cmake:    set( FFTW_MULTITHREAD TRUE )
clients/tests/cmake/FindFFTW.cmake:    set( FFTW_MULTITHREAD TRUE )
clients/tests/cmake/FindFFTW.cmake:    set( FFTW_MULTITHREAD TRUE )
clients/tests/cmake/FindFFTW.cmake:    set( FFTW_MULTITHREAD TRUE )
clients/tests/cmake/FindFFTW.cmake:  set( FFTW_MULTITHREAD TRUE )
clients/tests/gtest_main.cpp:#ifdef FFTW_MULTITHREAD
```
and

```

lamikr@ub2404:~/own/rock/src/sdk/rocm-libraries/projects/hipfft$ git grep FFTW_MULTITHREAD
clients/cmake/FindFFTW.cmake:    set( FFTW_MULTITHREAD TRUE )
clients/cmake/FindFFTW.cmake:    set( FFTW_MULTITHREAD TRUE )
clients/cmake/FindFFTW.cmake:    set( FFTW_MULTITHREAD TRUE )
clients/cmake/FindFFTW.cmake:    set( FFTW_MULTITHREAD TRUE )
clients/cmake/FindFFTW.cmake:  set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:      set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:      set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:      set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:      set( FFTW_MULTITHREAD TRUE )
clients/tests/CMakeLists.txt:if(FFTW_MULTITHREAD)
clients/tests/CMakeLists.txt:  target_compile_options( hipfft-test PRIVATE -DFFTW_MULTITHREAD )
clients/tests/gtest_main.cpp:#ifdef FFTW_MULTITHREAD
```